### ISSUE TITLE
memory access out of bounds (WASM) when loading large arrays 

### ISSUE THREAD

**Issue by andrea-dintino** [dyne/Zenroom#910]
This Zencode (in WASM only, in CLI it works fine): 

```
Given I have a 'base64 array' named 'array_of_mldsa44_signatures'
then print data

```

Run with the file attached [load-big-array.json](https://github.com/user-attachments/files/16390338/load-big-array.json)

- in Apiroom produces this:

```
RuntimeError: memory access out of bounds
    at 006c0846:0x50c6c
    at 006c0846:0xa9938
    at vA (main.1fe1c4af.js:2:1083407)
    at main.1fe1c4af.js:2:3455615
    at main.1fe1c4af.js:2:4357690
    at new Promise (<anonymous>)
    at DF (main.1fe1c4af.js:2:4357377)
    at async A (main.1fe1c4af.js:2:4360147)
```

- It also crashes the Slangroom editor

**Comment by matteo-cristino** [comment]
What version of zenroom are you using? This should have been solved by ba64eb1e478cfc70d67f7941fa839c492654d4da and include in version 4.32.5

**Comment by matteo-cristino** [comment]
From what I see:
* apiroom -> zenroom@4.32.2
* slangroom editor -> zenroom@4.31.2

Thus none of them contain the fix, we should try to update them and see if this still happens

**Comment by matteo-cristino** [comment]
This was indeed solved in zenroom 4.32.5. Testing now on apiroom with version 4.36.0 works prefectly as intended
